advanced-creation.md

```diff
@@ -111,3 +112,119 @@ const unicorn = got.create(settings);
 // Same as:
 const unicorn = got.extend({headers: {unicorn: 'rainbow'}});
 ```
+
+### Merging instances
```

Comment:
Can you add an entry to the Highlights section in the readme called `Composable`, which links to this section?